lucene/core/src/java/org/apache/lucene/index/IndexingChain.java

```diff
@@ -1313,4 +1259,110 @@ public void recycleIntBlocks(int[][] blocks, int offset, int length) {
       bytesUsed.addAndGet(-(length * (IntBlockPool.INT_BLOCK_SIZE * Integer.BYTES)));
     }
   }
+
+  private static final class FieldSchema {
```

Comment:
can you add a bit more docs, e.g. how this it differ from FieldInfo?

Reply:
Addressed in 6e7540ebd0ef79536cffabcf0ddc7a592b792252